website/www/site/content/en/documentation/io/built-in/snowflake.md

```diff
@@ -137,6 +154,303 @@ Snowflake IO library supports following options that can be passed via the [comm
 
 `--loginTimeout` Login timeout. Optional.
 
+`--snowPipe` SnowPipe name. Optional.
+
+In the connector code SnowflakePipelineOptions looks following (and it can be [extended](https://beam.apache.org/documentation/io/built-in/snowflake/#extending-pipeline-options)):
+{{< highlight java >}}
+public interface SnowflakePipelineOptions extends PipelineOptions, StreamingOptions {
```

Comment:
I'm not sure if we should keep the whole `SnowflakePipelineOptions` file in docs. Maybe it would be better to make a list + pointer to the file.

Reply:
TBD

Reply:
I agree, yes, we should remove this code (it was useful in previous documentation). If we want we can add the link as @purbanow suggested https://github.com/apache/beam/blob/master/sdks/java/io/snowflake/src/main/java/org/apache/beam/sdk/io/snowflake/SnowflakePipelineOptions.java

Reply:
I will fix it